.github/workflows/release.yml

```diff
@@ -0,0 +1,52 @@
+name: Create SDK release
+
+on: 
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: macOS-latest
+    
+    steps:
+    
+    - uses: actions/checkout@master
+      with:
+        ref: 'master'
+    
+    - uses: actions/setup-ruby@v1
+      with:
+        ruby-version: '2.6' 
+        
+    - name: Install dependencies
+      run: gem install cocoapods
+    
+    - name: Create a draft release
+      uses: actions/github-script@0.9.0
+      with:
+        script: |
+          // Get the current date
+          const today = new Date();
+          const yyyy = today.getFullYear();
+          const mm = String(today.getMonth() + 1).padStart(2, '0');
+          const api_version = yyyy + '-' + mm
+
+          github.repos.createRelease({
+            owner: context.repo.owner,
+            repo: context.repo.repo,
+            name: "ADD VERSION"
```

Comment:
What is the name here? Does this correspond to the title/name of the release? We want the format to be consistent with what we have today

Reply:
Whoever is drafting the release will need to change the name.